src/bin/vip-cache-purge-url.js

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env node
+
+/**
+ * @flow
+ * @format
+ */
+
+/**
+ * External dependencies
+ */
+
+/**
+ * Internal dependencies
+ */
+import { appQuery, purgeCache } from 'lib/api/cache-purge';
+import command from 'lib/cli/command';
+import { trackEvent } from 'lib/tracker';
+import { readFromFile } from 'lib/read-file';
+import * as exit from 'lib/cli/exit';
+
+const examples = [
+	{
+		usage: 'vip @123.production cache purge-url <URL>',
```

Comment:
We generally don't add the `@` shorthand to usage commands since it is provided separately by the echoed options help text

```suggestion
		usage: 'vip cache purge-url <URL>',
```